### ISSUE TITLE
Example of data cross-checking script

### ISSUE THREAD

**Issue by showerst** [openstates/people#14]
Just for fun and to test out the schema, i threw together a script that loops through our members, and cross-checks them against google Civic info API.

https://github.com/showerst/people/blob/vip/scripts/vip-crosscheck.py

I think this is the kind of thing we could set up to run once in a while and ping us about potential changes proactively.

**Comment by csnardi** [comment]
Perhaps this could also be made easier with something like automatic PRs or issue generation when a scraper encounters differing information from what's in this repo? I know the people/committee scrapers have been turned off for now, but it seems impractical to expect users to run the scrapers on their own to manually update the data.

**Comment by showerst** [comment]
@csnardi yeah I like this idea. Eventually we'll have to have a better system than "learn how to setup openstates, uncomment the people scrapers, fix an errors, run a local scrape, and then convert the files" as well, we just haven't had the hours to get things totally mature yet. Hopefully this hacky system with get us through january then we can spend some time improving the tooling significantly.
